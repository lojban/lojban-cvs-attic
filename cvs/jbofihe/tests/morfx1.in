#
# EXPERIMENTAL STUFF (word forms are non-conformant with standard lojban refgram.)

require "testpak.pl";

$prog="vlatai";

# Check basic scanning of lujvo with fu'ivla rafsi
$in = <<EOF;
gerzdanyrotfaila
rotfailybau
rotfailygerbau
rotfailygerkybau
rostailygerkybau
gerkyrotfaila
gerkyrotfailybau
geskyrotfailybau
gerkacyrotfailybau
geskacyrotfailybau
zdanyrotfailybau
xesyprenrapostolo
xesyprenrxapostolo
EOF
&test(stdin => $in);

# FIXME: Need to do more testing once DFA can recognize fu'ivla as 1st component of word.
# Particularly, need to work out the slinkui and tosmabru stuff in this context.
# Need some corner cases to test these.

#$in = <<EOF;
#EOF
#&test(stdin => $in);

# vim:ft=perl

