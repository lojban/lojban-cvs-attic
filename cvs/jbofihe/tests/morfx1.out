############################################################
PROGRAM : vlatai
========================================
STDIN :
----------
gerzdanyrotfaila
rotfailybau
rotfailygerbau
rotfailygerkybau
rostailygerkybau
gerkyrotfaila
gerkyrotfailybau
geskyrotfailybau
gerkacyrotfailybau
geskacyrotfailybau
zdanyrotfailybau
xesyprenrapostolo
xesyprenrxapostolo

-------------------------
========================================
Exit status : 0
Core dumped : NO
Signal      : 0
========================================
__STDOUT
----------
gerzdanyrotfaila          : UNMATCHED : gerzdanyrotfaila
rotfailybau               : UNMATCHED : rotfailybau
rotfailygerbau            : UNMATCHED : rotfailygerbau
rotfailygerkybau          : UNMATCHED : rotfailygerkybau
rostailygerkybau          : lujvo (with cultural rafsi) :  ro stailygerkybau
gerkyrotfaila             : UNMATCHED : gerkyrotfaila
gerkyrotfailybau          : UNMATCHED : gerkyrotfailybau
geskyrotfailybau          : UNMATCHED : geskyrotfailybau
gerkacyrotfailybau        : UNMATCHED : gerkacyrotfailybau
geskacyrotfailybau        : UNMATCHED : geskacyrotfailybau
zdanyrotfailybau          : UNMATCHED : zdanyrotfailybau
xesyprenrapostolo         : UNMATCHED : xesyprenrapostolo
xesyprenrxapostolo        : UNMATCHED : xesyprenrxapostolo
-------------------------
__STDERR
----------
-------------------------
